cmd/root_test.go

```diff
@@ -0,0 +1,80 @@
+package cmd
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/screwdriver-cd/sd-local/buildlog"
+	"github.com/screwdriver-cd/sd-local/config"
+	"github.com/screwdriver-cd/sd-local/launch"
+
+	"github.com/screwdriver-cd/sd-local/screwdriver"
+)
+
+type mockAPI struct{}
+type mockLogger struct{}
+type mockLaunch struct{}
+
+func (mock mockAPI) Job(jobName, filePath string) (screwdriver.Job, error) {
+	return screwdriver.Job{}, nil
+}
+
+func (mock mockAPI) JWT() string { return "" }
+
+func (mock mockLogger) Run() {}
+
+func (mock mockLogger) Stop() {}
+
+func (mock mockLaunch) Run() error { return nil }
+
+func setupTest() {
```

Comment:
How about to call `setupTest` in`TestMain()`?
https://golang.org/pkg/testing/#hdr-Main

Then you do not need to call  `setupTest()` at each Test function.

Also I think just `setup()` is cleaner.

Reply:
I fixed it